src/Microsoft.DotNet.Build.Tasks/PackageFiles/VersionTools.targets

```diff
@@ -66,22 +63,47 @@
+    Update to the latest dependencies available remotely, then submit a pull request.
+  -->
+  <Target Name="UpdateToRemoteDependenciesAndSubmitPullRequest">
+    <UpdateToRemoteDependenciesAndSubmitPullRequest DependencyBuildInfo="@(DependencyBuildInfo)"
+                                                    ProjectJsonFiles="@(ProjectJsonFiles)"
+                                                    UpdateStep="@(UpdateStep)"
+                                                    ProjectRepoName="$(ProjectRepoName)"
+                                                    ProjectRepoOwner="$(ProjectRepoOwner)"
+                                                    ProjectRepoBranch="$(ProjectRepoBranch)"
+                                                    GitHubAuthToken="$(GitHubAuthToken)"
+                                                    GitHubUser="$(GitHubUser)"
+                                                    GitHubEmail="$(GitHubEmail)"
+                                                    GitHubAuthor="$(GitHubAuthor)"
+                                                    NotifyGitHubUsers="@(NotifyGitHubUsers)"
+                                                    CurrentRefXmlPath="$(CurrentRefXmlPath)"
+                                                    AlwaysCreateNewPullRequest="$(AlwaysCreateNewPullRequest)"
+                                                    CommitMessage="$(CommitMessage)" />
+  </Target>
 
-  <Target Name="UpdateDependenciesAndSubmitPullRequest">
-    <UpdateDependenciesAndSubmitPullRequest DependencyBuildInfo="@(DependencyBuildInfo)"
-                                            ProjectJsonFiles="@(ProjectJsonFiles)"
-                                            UpdateStep="@(UpdateStep)"
-                                            ProjectRepoName="$(ProjectRepoName)"
-                                            ProjectRepoOwner="$(ProjectRepoOwner)"
-                                            ProjectRepoBranch="$(ProjectRepoBranch)"
-                                            GitHubAuthToken="$(GitHubAuthToken)"
-                                            GitHubUser="$(GitHubUser)"
-                                            GitHubEmail="$(GitHubEmail)"
-                                            GitHubAuthor="$(GitHubAuthor)"
-                                            NotifyGitHubUsers="@(NotifyGitHubUsers)"
-                                            CurrentRefXmlPath="$(CurrentRefXmlPath)"
-                                            AlwaysCreateNewPullRequest="$(AlwaysCreateNewPullRequest)" />
+  <!--
+    Deprecated auto-upgrade PR target. Redirects to the target's new name.
+  -->
+  <Target Name="UpdateDependenciesAndSubmitPullRequest"
+          DependsOnTargets="UpdateToRemoteDependenciesAndSubmitPullRequest">
+    <Message Text="'UpdateDependenciesAndSubmitPullRequest' has been renamed to 'UpdateToRemoteDependenciesAndSubmitPullRequest'. The old name is deprecated. Ran the new target instead."
```

Comment:
(nit) `Ran the new target instead` => `Run`

Reply:
This is intended to describe the automatic redirect MSBuild did with `DependsOnTargets` rather than tell the user what to do. Maybe I should use `CallTarget` and say `Running` before to make it clearer.